src/decrypt-api/Startup.cs

```diff
@@ -80,6 +80,7 @@ public Startup(IHostingEnvironment env)
                             Configuration.GetValue<int>("KeyManagement:KeyVault:MaximumDataLength"));
                     case "AESKey":
                         var key = Configuration.GetValue<string>("KeyManagement:AES:Key");
+                        Console.WriteLine($"AES key used: {key}");

```

Comment:
Why you don't use the logger?

Reply:
I think this is called before the logger is initialized